data/notebooks/explore.py

```diff
@@ -37,6 +54,8 @@ def search_activity(change):
     limit = change.new if change.owner is w_limit else w_limit.value
     w_activity.value = None
     db = bw2data.Database(database)
+    if not db:
+        return
```

Comment:
nit: I'd rather raise with an explicit message here

Reply:
We shouldn't raise here since we are in a GUI. The function is a widget change event handler. If we have no db, we just don't handle anything.